### DISCUSSION TITLE
ESP32 16 CHANNEL PWM

### DISCUSSION THREAD

**Discussion by koko009** [arendst/Tasmota#19999] [Q&A]
i want help 
i need 10channel pwm light strip solid color 
tasmota version latest 
devices esp32 
now my web ui limit 5 channel only 
can any one help me how to config for more then 5 channel pwm ?

**Comment by sfromis** [comment]
As the Tasmota lights feature is generally used for PWM, you get no more than 5 PWM channels this way.

Still, you can address the addtional PWM channels using the `PWM` command (not in the web UI), or using the Berry language.
